lib/membrane_rtc_engine/tees/tee.ex

```diff
@@ -175,17 +178,16 @@ defmodule Membrane.RTC.Engine.Tee do
         _ctx,
         state
       ) do
-    %{current_variant: variant} = state.routes[pad]
+    %{current_variant: current_variant} = state.routes[pad]
 
-    if Map.has_key?(state.inactive_variants, variant) do
-      Membrane.Logger.debug("""
-      Endpoint #{endpoint_id} requested keyframe for inactive track variant: #{variant}. \
-      Track #{inspect(state.track)}. Ignoring.
```

Comment:
Shouldn't we still have this log for situations like this?

Reply:
It's no longer possible. When we receive TrackVariantPaused we are resetting current_variant